atom/renderer/atom_renderer_client.cc

```diff
@@ -34,6 +35,21 @@ bool IsDevToolsExtension(content::RenderFrame* render_frame) {
       .SchemeIs("chrome-extension");
 }
 
+v8::Local<v8::Value> GetHiddenValue(v8::Local<v8::Context> context,
```

Comment:
I think we should expose the `GetHiddenValue` helper from `atom_api_v8_util.cc` (by adding a public `atom_api_v8_util.h` header), and then call the function here instead of adding a new copy for every file.